src/views/Pools/components/CardFooter.tsx

```diff
@@ -94,6 +114,10 @@ const CardFooter: React.FC<Props> = ({
   const blocksUntilStart = Math.max(startBlock - block, 0)
   const blocksRemaining = Math.max(endBlock - block, 0)
 
+  const isMetamask = (window as any).ethereum && (window as any).ethereum.isMetaMask && tokenAddress
```

Comment:
Please add these types in `react-app-env.d.ts`. You can see an example [here](https://github.com/pancakeswap/pancake-swap-interface/blob/598fece7e55686726728581bcb8c8bdfaaeae994/src/react-app-env.d.ts#L11)